WolvenKit.App/ViewModels/Shell/AppViewModel.WScript.cs

```diff
@@ -76,6 +76,19 @@ private async Task RunFileValidationOnProject()
             return;
         }
 
+        var txtFilesInResources = ActiveProject.ResourceFiles.Where(f => f.EndsWith(".txt"))
+            .Select(f => ActiveProject.GetRelativeResourcePath(f).GetResolvedText())
```

Comment:
Why is it need to get the relative resource path if we only check the extension?

Reply:
For the output below